uint8 state
uint16[5] channels

uint8 STATE_NOT_FOUND = 1
uint8 STATE_ERROR = 2
uint8 STATE_RC = 3
uint8 STATE_SERIAL = 4
